src/Api/Controllers/AuthController.php

```diff
@@ -41,7 +42,7 @@
  * @property \Baka\Mail\Message $mail
  * @property Apps $app
  */
-class AuthController extends \Baka\Auth\AuthController
+class AuthController extends \Canvas\Auth\AuthController
```

Comment:
@rwhite27 should we move the controllers to traits since they are withing the same library?